Allwmake

```diff
@@ -123,20 +125,19 @@ log "  ADAPTER_GLOBAL_LIBRARY_PATHS    = ${ADAPTER_GLOBAL_LIBRARY_PATHS}"
 
 # Run wmake (build the adapter) and check the exit code
 log ""
-log "Building with WMake (see the wmake.log log file)..."
-wmake ${ADAPTER_WMAKE_OPTIONS} libso > wmake.log 2>&1
-log ""
-if [ $? -ne 0 ] || [ "$(grep -c -E "error:" wmake.log)" -ne 0 ]; then
+log "Building with WMake (see the wmake.log log file)...\n"
+if ! wmake "${ADAPTER_WMAKE_OPTIONS}" libso >wmake.log 2>&1 ||
```

Comment:
Now I have finally the chance to comment on this: I don't like the hidden compiler output and comparing it to other libraries, e.g. swak4foam, it seems to be non-default hiding it. Maybe it is also a developer problem, but could we please pipe the compiler output (additionally) to `cout` by default. If something goes wrong, one would directly see.

Reply:
I changed this to
```shell
if ! wmake "${ADAPTER_WMAKE_OPTIONS}" libso 2>&1 | tee wmake.log ||
    [ "$(grep -c -E "error:" wmake.log)" -ne 0 ]; then
```
This way, we get both the verbose output and the `wmake.log` file.

Since this throws a few known deprecation warnings to the user, I added some inline comments explaining that these are intentionally there.